build.xml

```diff
@@ -61,6 +61,14 @@
     </java>
   </target>
 
+  <target name="runsky-firmware" depends="jar"
+          description="run MSPSim with platform Sky">
+    <java fork="true" classpath="${jarfile}"
+          classname="se.sics.mspsim.platform.sky.SkyNode" failonerror="true">
+      <arg line="${ARGS}"/>
+    </java>
+  </target>
+
```

Comment:
What is the advantage of this new target compared to the normal target?

> ant runsky -DFIRMWAREFILE=firmware/sky/cputest.sky

Reply:
I wasn't aware it worked that way, this isn't needed.